homeworks/serhii.noskov_SerhiiNSN/7-arrays-loops/arrays.js

```diff
@@ -0,0 +1,76 @@
+const arrayOne = [3, 0, -5, 1, 44, -12, 3, 0, 0, 1, 2, -3, -3, 2, 1, 4, -2, -3, -1];
+const arrayTwo = [-1, -8, -2];
+const arrayThree = [1, 7, 3];
+const arrayFour = [1, undefined, 3, 5, -3];
+const arrayFive = [1, NaN, 3, 5, -3];
+
+function maxNumberOfArr(array) {
+    let sum = array[0];
+    for (let i = 0; i < array.length; i++) {
+        if (array[i] > sum) {
+            sum = array[i];
+        }
+    }
+    return sum;
+}
+
+console.log(maxNumberOfArr(arrayOne)); // 44
+console.log(maxNumberOfArr(arrayTwo)); // -1
+console.log(maxNumberOfArr(arrayThree)); // 7
+console.log(maxNumberOfArr(arrayFour)); // 5
+console.log(maxNumberOfArr(arrayFive)); // 5
+
+function minNumberOfArr(array) {
+    let sum = array[0];
+    for (let i = 0; i < array.length; i++) {
```

Comment:
```suggestion
    for (let i = 1; i < array.length; i++) {
```
If you set the `array[0]` as an init value, you can start from the next element